.github/workflows/test.yml

```diff
@@ -17,6 +17,8 @@ jobs:
         fetch-depth: 0
         # fetch PR commit, not predicted merge commit
         ref: ${{ github.event.pull_request.head.sha }}
+        # This is needed to be able to push and trigger CI with that push
+        token: ${{ secrets.NIV_UPDATER_TOKEN }}
```

Comment:
I'll try without this. But my hunch is that this activates the checkout over `ssh`, which can push securely.

Reply:
It worked! Maybe zap it from `.github/workflows/update-hash.yml` too?